README.md

```diff
@@ -1,2 +1,33 @@
-# awesome-innersource
-Awesome InnerSource (per [sindresorhus/awesome](https://github.com/sindresorhus/awesome/blob/main/awesome.md))
+# Awesome InnerSource [![Awesome](https://awesome.re/badge-flat2.svg)](https://awesome.re)
+
+<!--lint ignore double-link-->
+A curated list of awesome **[InnerSource](https://innersourcecommons.org/)** resources (books, articles, blogs, projects and more). 
+
+<!--lint ignore double-link-->
+[<img src="assets/images/innersource-logo.png" align="right" width="100" alt="Apache JMeter">](https://innersourcecommons.org/)
+
+<!--lint ignore double-link-->
+> [InnerSource](https://innersourcecommons.org/) is a software development strategy that applies open source best practices to proprietary code. InnerSource can help establish an open source culture within an organization while retaining software for internal use. Teams use InnerSource to increase visibility, strengthen collaboration and break down silos. Through the set of practices, InnerSource helps companies to simplify, standardize and make cost savings globally and across departments.
```

Comment:
Just curious: Where is this definition from?

Reply:
Partially from https://about.gitlab.com/topics/version-control/what-is-innersource/ and partially from my company. Let me know if you have some better candidate!

Reply:
Not at all. I just had not heard this definition before, so I figured it was a mix of multiple resources.
Can certain stay this way!